### PR TITLE
Fix sql statement 

### PR DESCRIPTION
**Pull request by mightymop**
Fix for: https://github.com/igniterealtime/openfire-monitoring-plugin/issues/47

### CODE REVIEW

File: changelog.html [hunk 1]
```diff
@@ -43,6 +43,12 @@
 <h1>
 Monitoring Plugin Changelog
 </h1>
+    
+<p><b>1.8.2</b> -- November 25, 2019</p>
+<ul>
+    <li>[<a href='https://github.com/igniterealtime/openfire-monitoring-plugin/issues/60'>Issue #53</a>] - Java11 ClassNotFoundException when deploying.</li>
+    <li>Fix Syntax in SQL Statement for SQL Server in PaginatedMucMessageQuery</li>
```
Comment (guusdk) [inline changelog.html]: I hate to nag, but can you please provide a link (`a href=`) to the issue on github?

Comment (mightymop) [reply changelog.html]: of cource.. wait a moment ... done

File: changelog.html [hunk 2]
```diff
@@ -43,6 +43,12 @@
 <h1>
 Monitoring Plugin Changelog
 </h1>
+    
+<p><b>1.8.2</b> -- November 25, 2019</p>
+<ul>
+    <li>[<a href='https://github.com/igniterealtime/openfire-monitoring-plugin/issues/60'>Issue #53</a>] - Java11 ClassNotFoundException when deploying.</li>
```
Comment (guusdk) [inline changelog.html]: This adds issue 53 to the changelog (while providing a link to issue 60), which isn't part of this PR. I'll merge the PR as-is, and manually correct this later.

**Comment by guusdk** [comment]
Thank you for your contribution, and your patience :)

**Pull request merged**